tests/functional/delete_runs/test_delete_runs.py

```diff
@@ -0,0 +1,124 @@
+#
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" delete_runs function test.
+
+WARNING!!!
+This is a generated test skeleton remove the parts not required by the tests.
+WARNING!!!
+
+"""
+import os
+import unittest
+import itertools
+import subprocess
+from datetime import datetime
+
+from libtest import env
+
+
+def run_cmd(cmd):
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    proc.communicate()
+    return proc.returncode
+
+
+class TestSkeleton(unittest.TestCase):
+
+    def setUp(self):
+        # TEST_WORKSPACE is automatically set by test package __init__.py .
+        test_workspace = os.environ['TEST_WORKSPACE']
+
+        test_class = self.__class__.__name__
+        print('Running ' + test_class + ' tests in ' + test_workspace)
+
+        self._cc_port = env.import_test_cfg(test_workspace)[
+            'codechecker_cfg']['viewer_port']
+
+        # Get the test project configuration from the prepared test workspace.
+        self._testproject_data = env.setup_test_proj_cfg(test_workspace)
+        self.assertIsNotNone(self._testproject_data)
+
+        # Setup a viewer client to test viewer API calls.
+        self._cc_client = env.setup_viewer_client(test_workspace)
+        self.assertIsNotNone(self._cc_client)
+
+        # Get the CodeChecker cmd if needed for the tests.
+        self._codechecker_cmd = env.codechecker_cmd()
+
+    def test_multiple_deletion(self):
```

Comment:
I would add some comment or make the test method more descriptive that this is a command line test.